docs/setuptools.txt

```diff
@@ -1655,17 +1655,26 @@ Distributing Extensions compiled with Cython
 --------------------------------------------
 
 ``setuptools`` will detect at build time whether Cython is installed or not.
-If Cython is not found ``setputools`` will ignore pyx files. In case it's
-available you are supposed it will work with just a couple of adjustments.
-``setuptools`` includes transparent support for building Cython extensions, as
-long as you define your extensions using ``setuptools.Extension``.
-Then you should use Cython own ``build_ext`` in ``cmdclass``, e.g.::
+If Cython is not found ``setputools`` will ignore pyx files.
 
-   from Cython.Distutils import build_ext
+To ensure Cython is available, include Cython in the build-requires section
+of your pyproject.toml::
```

Comment:
Would be nice to link this to something that shows how to use `pyproject.toml`, since it's not obvious what goes in place of the `...`, but we also don't want to distract from the important part in this example.

Reply:
Yeah, I was trying not to be too prescriptive, but at the same time, it's probably important that 'setuptools' and 'wheel' are included there, as defining the build deps overrides the defaults (IIUC).